src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml

```diff
@@ -41,6 +42,10 @@
             <argument type="service" id="payum" />
         </service>
 
+        <service id="sylius.behat.email_checker" class="%sylius.behat.email_checker.class%">
```

Comment:
If we are moving this checker out of behat namespace, we should remove `behat` keyword from service name.